### ISSUE TITLE
Provide guidance for BNS preorder and register function calls

### ISSUE THREAD

**Issue by nanonerd** [leather-io/extension#1503]
Hi. I PREORDER a btc domain name:  hartford. It was eventually confirmed.

Here is the PREORDER transaction:  https://explorer.stacks.co/txid/13dc198dc4654d117bb91692a02e5a765290aaeb6e57e70f76caa8a00f73bf3a?chain=mainnet

I tried to REGISTER (reveal to network). It failed a few times. I can't remember the error. Hours later, I tried again and now get a "Broadcast error: "transaction rejected - TooMuchChaining". 

I uploaded 3 images. #2 and #3 show the REGISTER failure message. Please let me know if you need any other info. Thanks.

![hartford1](https://user-images.githubusercontent.com/5571462/127480620-a294adad-cb4c-4eb6-a6b3-8f581d29b74c.png)
![hartford2](https://user-images.githubusercontent.com/5571462/127480634-8f50c22d-17a8-42d5-9d47-86baf370f49e.png)
![hartford3](https://user-images.githubusercontent.com/5571462/127480642-d42da1f1-42c5-466e-938f-363af869114e.png)

**Comment by markmhendrickson** [comment]
You may have encountered this problem in particular: https://github.com/blockstack/stacks-wallet-web/issues/1498

**Comment by nanonerd** [comment]
Hi, anything I can do to fix? Or do I have to wait for an upgrade with a potential fix?
If it may be stuck for awhile, that is Ok. I am not too pressed about this domain (hartford). But since I did PREORDER with success, it would be nice to REGISTER it at some point. Thanks.

**Comment by nanonerd** [comment]
Hi, I got another account error during REGISTER: "transaction rejected - TooMuchChaining". 

Stacks Account: SP3V0J1MSE4N322YYT95B6J4SMCNKF1KKDX8BRXR

Here is the PREORDER confirmation: https://explorer.stacks.co/txid/d31ae7adbde58ad33f848bfd19c45dfb244868ded99e50d949064175a244380f?chain=mainnet

QUESTION: Given that the PreORDER succeeded, technically, I have control of the domain name, right? FYI: It still shows as available if you search for it (e.g., "hartford"). But I assume if someone else tries to PREORDER, it will fail for them. Is this correct?

**Comment by markmhendrickson** [comment]
> I assume if someone else tries to PREORDER, it will fail for them. Is this correct?

I believe that's correct, though @aulneau may be a good person to confirm.

If you uninstall and reinstall the extension, do you still get a TooMuchChaining error?

**Comment by aulneau** [comment]
That is not correct. You do not own a domain just by preordering it. You must register it to own it.

**Comment by nanonerd** [comment]
@markmhx - Hi Mark. Thanks for the suggestion. I removed and reinstalled the web wallet extension. That helped because when I REGISTER, instead of constant "chaining error" msg, this time it "failed" and showed the START OVER button. - I did a new PREORDER + REGISTER and I am good now. 

My issue is resolved (with workaround). But since the original cause of the "chaining error" is still unknown, I will defer to you whether to close this ticket or not. Thanks!

**Comment by nanonerd** [comment]
@aulneau - here is what the UI shows now. As is, it is not totally clear at what point I get control of the domain name. 

**PREORDER (reserves the name)** [_this sounds like I own it at this point_]

**REGISTER (reveals the name to the network)** 

Suggestion> Maybe adding something like "You must PREORDER + REGISTER to own the domain" will make it easier to understand. My 2 sats ... :-)

**Comment by markmhendrickson** [comment]
@nanonerd Thanks for the updates and suggestions. I'm glad to hear you're unblocked now.

I've modified this issue to track a possible enhancement in which we provide guidance in the UI as to how `name-preorder` and `name-register` functions behave from a UX perspective, so other users don't run into the same confusion.
